tools/i-pi/README.md

```diff
@@ -0,0 +1,18 @@
+This folder used to contain a bundled version of [i-PI](https://ipi-code.org)
+but the bundled version was removed in 05/2024.
```

Comment:
likely 06/2024 to be pedantic ^_^

Reply:
done

Reply:
Does this mean I can't merge this PR until next week? 😆

Reply:
You people are too efficient, I clearly underestimated you ^_^'

Reply:
@stanmoore1 it also means that you have to do a feature release within the next month. :wink: